enclave_wrangler/objects_api.py

```diff
@@ -295,15 +296,14 @@ def csets_and_members_enclave_to_db(
     print('Fetching new data from the N3C data enclave...')
     t0 = datetime.now()
     csets_and_members: Dict[str, List[Dict]] = fetch_cset_and_member_objects(since)
-
     print(f'  - Fetched new data in {(datetime.now() - t0).seconds} seconds:\n    OBJECT_TYPE: COUNT\n' +
           "\n".join(['    ' + str(k) + ": " + str(len(v)) for k, v in csets_and_members.items()]))
     return csets_and_members_to_db(con, schema, csets_and_members)
 
 
 def fetch_cset_and_member_objects(
-    since: Union[datetime, str], verbose=False
-) -> Dict[str, List[Dict]] :
+    since: Union[datetime, str] = '', cset_versions: List[Dict] = [], verbose=False
```

Comment:
@hugwuoke @Sigfried I updated `fetch_cset_and_member_objects()` so that you can either pass `since` or a list of cset version objects (I could have also went with allowing for a list of codeset_ids instead). I think there are a variety of cases where this will be useful. Where this will most immediately be useful for me is with unit testing. I'm going to write a test where I pass it a concept set that I know is going to fail because it has more members/items than the enclave will allow us to fetch. Then it will execute the rest of the code I'm writing to do our workaround to fetch that data.